extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItemExtractor.java

```diff
@@ -3,59 +3,93 @@
 import org.schabi.newpipe.extractor.InfoItemExtractor;
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeCommentsInfoItemExtractor;
 import org.schabi.newpipe.extractor.stream.StreamExtractor;
+import org.schabi.newpipe.extractor.utils.Utils;
 
 import javax.annotation.Nullable;
 
 public interface CommentsInfoItemExtractor extends InfoItemExtractor {
 
     /**
-     * Return the like count of the comment, or -1 if it's unavailable
+     * Return the (approximate) like count of the comment, or -1 if it's unavailable<br/>
+     * NOTE: Currently only implemented for YT {@link YoutubeCommentsInfoItemExtractor#getLikeCount()}
+     * with limitations
      *
      * @see StreamExtractor#getLikeCount()
      */
-    int getLikeCount() throws ParsingException;
+    default int getLikeCount() throws ParsingException {
+        return -1;
+    }
+
+    /**
+     * The formatted text (e.g. 420, 4K, 4.2M) of the votes<br/>
+     * May be language dependent
+     */
+    default String getTextualVoteCount() throws ParsingException {
```

Comment:
change the interface to `getTextualLikeCount()`. In YouTube, vote count IS like count, but other services may have both like and dislike count, and vote count would be not precise